.pre-commit-config.yaml

```diff
@@ -11,3 +11,11 @@ repos:
     hooks:
     - id: ruff
       args: [--fix]
+  - repo: https://github.com/python-poetry/poetry
+    rev: '1.4.0'  # add version here
+    hooks:
+      - id: poetry-check
+      - id: poetry-lock
```

Comment:
The github check timed out on the poetry lock step, but it ran fine locally... is there any way to skip hooks on pull requests, but run them locally?

Reply:
```
ci:
    skip: [poetry-lock, poetry-export]
```
Something like that, see the skip section here: https://pre-commit.ci/#configuration

Reply:
Ah grand, thanks!